lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -674,13 +658,7 @@ export class UserAgentApplication {
         // populate QueryParameters (sid/login_hint/domain_hint) and any other extraQueryParameters set by the developer
         serverAuthenticationRequest = this.populateQueryParams(account, request, serverAuthenticationRequest);
 
-        // Cache nonce
-        this.cacheStorage.setItem(Constants.nonceIdToken, serverAuthenticationRequest.nonce, this.inCookie);
-        serverAuthenticationRequest.state = serverAuthenticationRequest.state;
-
-        // Cache account and authority
-        this.setAccountCache(account, serverAuthenticationRequest.state);
-        this.setAuthorityCache(serverAuthenticationRequest.state, acquireTokenAuthority.CanonicalAuthority);
```

Comment:
it's canonicalAuthority here, however in the updateCacheEntries is using authority